Makefile

```diff
@@ -6,3 +6,15 @@ SHELL := /usr/bin/env bash
 .PHONY: boilerplate-update
 boilerplate-update:
 	@boilerplate/update
+
```

Comment:
Can we avoid the Makefile changes? We try to keep the Makefile as minimal as possible.

Reply:
Change reverted. As discussed, will add the change via boilerplate instead.